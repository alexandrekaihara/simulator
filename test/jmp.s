JMP MUSTJUMP
OUTPUT MINUSONE
STOP
MUSTJUMP: OUTPUT ONE
STOP
ONE: CONST 1
MINUSONE: CONST -1