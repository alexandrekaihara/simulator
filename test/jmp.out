5 5 13 9 14 13 8 14 1 -1